components/user-store/org.wso2.carbon.identity.user.store.count/src/main/java/org/wso2/carbon/identity/user/store/count/UserStoreCountService.java

```diff
@@ -202,4 +173,58 @@ public Long countByClaimInDomain(String claimURI, String valueFilter, String dom
 
     }
 
+    private RealmConfiguration realmConfiguration = null;
+
+    /**
+     * Get User count.
+     *
+     * @param claimURI claim uri
+     * @param valueFilter filter
+     * @return user count
+     * @throws UserStoreCounterException UserStoreCounterException
+     */
+    private long getUserCountWithClaims(String claimURI, String valueFilter) throws UserStoreCounterException {
+
+        try {
+            realmConfiguration = CarbonContext.getThreadLocalCarbonContext().getUserRealm().getRealmConfiguration();
+            return UserStoreCountDSComponent.getRealmService().getUserRealm(this.realmConfiguration).
+                    getUserStoreManager().getUserCountWithClaims(claimURI, valueFilter);
+
+        } catch (UserStoreException e) {
+            String ErrorMsg = "No user count is retrieved from the user store.";
```

Comment:
How can you say the error is "No user count is retrieved from the user store."?
What you can say here is you get "this" error while doing "this" operation.